packages/spark-core/_spark-core.scss

```diff
@@ -45,6 +45,7 @@
 @import './base/forms';
 @import './base/links';
 @import './base/typography';
+@import './components/alerts';
```

Comment:
We changed notifications to be called Alerts. Notifications will be a new pattern.